rjc_stock/views/stock_move_views.xml

```diff
@@ -6,7 +6,7 @@
         <field name="inherit_id" ref="stock.view_stock_move_line_operation_tree"/>
         <field name="arch" type="xml">
             <field name="lot_id" position="attributes">
-                <attribute name="options">{'create': True}</attribute>
+                <attribute name="options">{'create': False}</attribute>
```

Comment:
@kranokporn  This is not show create/edit in dropdown many2one field, Am I correct ?

Reply:
@newtratip I think that's right, this is not show in lot_id field. Option is boolean false.

Reply:
OK